docs/docs/tutorial/android.md

```diff
@@ -0,0 +1,19 @@
+---
+sidebar_position: 9
+sidebar_label: Targetting Android
+title: Targetting Android
+---
+
+This tutorial explains how to set up the plugin to target Android devices.
+
+## Prerequisites
+- JDK 17 or later
+
+## Setup
+1. Follow the [installation](installation) instructions for your development platform.
+2. Make sure that the Unreal Editor is closed.
+3. Download a plugin [release](https://github.com/DolbyIO/comms-sdk-unreal/releases) for Android and unpack the plugin to the `Plugins` folder. The release version should match your development platform release. Say yes to any overwrite prompts.
+4. Edit `{UnrealEngineRoot}/Build/Android/Java/gradle/gradle/wrapper/gradle-wrapper.properties` and modify `distributionUrl` to `https\://services.gradle.org/distributions/gradle-7.4.1-all.zip`
+5. Open your project in Unreal Editor.
+6. Set `Project Settings -> Platforms -> Android SDK -> Location of JAVA` to the directory containing the required JDK version.
+7. Tick the `Project Settings -> Platforms -> Android -> Advanced Build -> Disable libc++_shared dependency validation in all dependencies` checkbox.
```

Comment:
Same as above, except here I strongly feel that the Android libs should be build with static libc++ just like the C++ SDK libs are for Ubuntu with clang10. I don't think this requirement should exist as it disables warnings for the project which may be pertinent.

Reply:
The only reason is dvc and even there adding statically the libc++ will probably introduce memory management issues when it comes to unreal loading it's own and that dealing with memory ownership will be colliding.

In our case, the dynamic version of libc++ is normally ok as it happens that it's the only thing they did right out of the box : the ndk version is quite new (!)